cirq/protocols/json_serialization.py

```diff
@@ -447,12 +447,11 @@ class SerializableByKey(SupportsJSON):
     """Protocol for objects that can be serialized to a key + context."""
 
     @doc_private
-    def _serialization_key_(self) -> str:
-        """Returns a unique string identifier for this object.
+    def _serialization_name_(self) -> str:
+        """Returns a string identifier for this object.
 
-        This should only return the same value for two objects if they are
-        equal; otherwise, an error will occur if both are serialized into the
-        same JSON string.
+        This identifier need not be globally unique; a unique suffix will be
+        dynamically generated to create the serialization key for this object.
```

Comment:
Mention recommended readability for the name

Reply:
Done.